File: Core/Entities/TodoTask.cs
namespace Core.Entities;

public class TodoTask : BaseEntity
{ 
    public string Title { get; set; } = null!;

    public string Text { get; set; } = null!;

    public Guid ProjectId { get; set; }

    public Project Project { get; set; } = null!;
    public int StatusId { get; set; }
    public TodoTaskStatus Status { get; set; } = null!;
    public Guid? EmployeeId { get; set; }
    public Employee? Employee { get; set; }
}

File: client/src/app/shared/models/TodoTask.ts
import Employee from './Employee';
import AvailableTaskStatus from './enums/AvailableStatus';

export default interface TodoTask {
  id: string;
  title: string;
  text: string;
  status: AvailableTaskStatus;
  employee: Employee | null;
  projectId: string;
}
